### ISSUE TITLE
Negative sign in non-math mode still wrapped in math.equation

### ISSUE THREAD

**Issue by danvim** [Mc-Zen/zero#113]
I noticed this issue only because I enabled `PDF/UA-1` mode, where my document with `#num("-123.4", math:false)` failed the check, as the negative sign is wrapped in an equation tag without alt text. If you look at the screenshort comparison, you can see the fonts aren't the same.

I would expect math symbols to all use body font in non-math mode. Or is it intentional that only digits would appear in body font?

<img width="1528" height="712" alt="Image" src="https://github.com/user-attachments/assets/a81d518c-4044-4d7c-a1cc-3c93154e2a18" />

```typ
#import "@preview/zero:0.7.0": num

#set text(font: "DejaVu Sans", fill: green)

#show text: it => context {
  if (text.font == "dejavu sans") {
    set text(fill: red)
    it
  } else {
    it
  }
}

-123.4

#num("-123.4", math:false)

#context {
  let elements = query(selector(math.equation))
  
  // Print the structural representation as raw text
  repr(elements)
}
```

**Comment by Mc-Zen** [comment]
Hi @danvim ,  thank you very much for reporting!

This is not intentional and I have no idea why the sign is still wrapped in an equation

**Comment by Mc-Zen** [comment]
oh actually this has been fixed along with #109 through #110 . This will be part of a release that I'm planning to publish soon.
